cassiopeia/solver/MaxCutSolver.py

```diff
@@ -0,0 +1,100 @@
+"""
+"""
+import itertools
+import networkx as nx
+import numpy as np
+import pandas as pd
+from typing import Callable, Dict, List, Optional, Tuple
+
+from cassiopeia.solver import GreedySolver
+from cassiopeia.solver import graph_utilities as g_utils
+
+
+class MaxCutSolver(GreedySolver.GreedySolver):
+    def __init__(
+        self,
+        character_matrix: pd.DataFrame,
+        missing_char: str,
+        meta_data: Optional[pd.DataFrame] = None,
+        priors: Optional[Dict] = None,
+        sdimension: Optional[int] = 3,
+        iterations: Optional[int] = 50,
+        weights: Optional[Dict] = None,
+    ):
```

Comment:
I still don't want to accept this into the codebase until it's better documented and tested. Please remove this from the PR.